rendermodules/rough_align/apply_rough_alignment_to_montages.py

```diff
@@ -61,39 +66,157 @@
 
 logger = logging.getLogger()
 
+
+def get_mask_paths(
+        mask_input_dir,
+        tilespecs,
+        read_masks_from_lowres_stack,
+        exts=['png', 'tif']):
+
+    results = {}
+
+    if read_masks_from_lowres_stack:
+        for t in tilespecs:
+            if t.ip[0].maskUrl is not None:
+                results[t.tileId] = t.ip[0].maskUrl
+
+    elif mask_input_dir is not None:
+        tids = [t.tileId for t in tilespecs]
+        maskfiles = []
+        for ext in exts:
+            e = mask_input_dir + '/*.' + ext
+            maskfiles += glob.glob(e)
```

Comment:
I agree... We cannot assume that masks will be stored in the same directory. We can either document this assumption here and revisit later to fix it.

Reply:
- just FYI, the file `apply_rough_alignment_to_montages.py` is not part of this PR. It is showing up because master got ahead of develop (my bad).
- nonetheless, good point, I'll change this. Point of reference, for the 20k sections, we had about fewer than 50 masks... not sure how much we need to worry about performance issues here.
- @gayathri, it is implicit from the schema that there is one mask directory. If there are multiple directories, we can run the module multiple times. Also, it is likely we won't be reading from directory at this point as we had added `read_masks_from_lowres_stack` as an option and the whole PR here is to get the masks into the lowres stack so we can use that.